articles/introduction.html

```diff
@@ -1,4 +1,4 @@
-﻿<h1>An Introduction Of Go</h1>
+<h1>An Introduction Of Go</h1>

```

Comment:
needs a lower case "of" here? to be consistent with the one in 101.html?

Reply:
Yes - very correct!

Reply:
Do you prefer me to type it for You, or shall I update my fork?

Reply:
Joking aside - I'd like us to find some easy and efficient way to improve Your texts together.

E.g. different with e.g. [#36](https://github.com/go101/go101/issues/36)  I don't consider a simple type worth an explicit issue and went straight into proposing textual changes.

Reply:
some people may be not familiar with PR system yet. So the issue way is acceptable.

You can make "Of" -> "of" change so that we can save a commit.

Reply:
btw: How about
<h1>Introducing <code>Go</code> (the language)</h1>

<meta>I prefer action words (= verbs) over nouns substituting for (and thereby 'freezing') lively processes.</meta>